command.go

```diff
@@ -338,7 +338,10 @@ func getTheme(s string) (Theme, error) {
 }
 
 func getNamedTheme(s string) (Theme, error) {
-	theme, suggestions, ok := findTheme(s)
+	theme, suggestions, ok, err := findTheme(s)
```

Comment:
Do we need both `ok` and `err`? I assume `err != nil` is equivalent to `ok`?

Reply:
There's a difference, even though we could unify the `bool` and `err` by using a typed error:

The `err` indicates a fatal error, e.g. when `themes.json` is corrupted. But there's also the situation where we just can't find the requested theme: in that case the `bool` is false, and we return suggestions.

Reply:
Gotcha, yeah that sounds good then.